### ISSUE TITLE
svg-img-alt failing when it shouldn't

### ISSUE THREAD

**Issue by tylerthehaas** [nickcolley/jest-axe#136]
The following sandbox is producing an svg with a title as a child element to the svg however jest-axe is failing the test. This should pass as it meets the accessibility spec.

https://codesandbox.io/s/relaxed-satoshi-0p2jz

**Comment by landonalder** [comment]
I'm having this same issue
